src/ring_theory/ideal/local_ring.lean

```diff
@@ -371,6 +371,32 @@ lemma map_comp (f : T →+* R) (g : R →+* S) [is_local_ring_hom f] [is_local_r
   (local_ring.residue_field.map g).comp (local_ring.residue_field.map f) :=
 ideal.quotient.ring_hom_ext $ ring_hom.ext $ λx, rfl
 
+section
+
+open ring_equiv local_ring
+
+@[simp]lemma map_id_apply {x : residue_field R} : map (ring_hom.id R) x = x :=
```

Comment:
Instead of marking this lemma simp, can you mark `map_id` above `simp`?

Reply:
Hi, thank you. Could you please elaborate?

Reply:
I mean put `@[simp]` on the existing `map_id` lemma that is a few lines higher in the file

Reply:
OK, thank you.